src/renderer/NumbersCellRenderer.ts

```diff
@@ -34,6 +37,11 @@ export default class NumbersCellRenderer extends ANumbersCellRenderer {
 
     return (ctx: CanvasRenderingContext2D, data: number[], offset: number, rowHeight: number) => {
       data.forEach((d: number, j: number) => {
+        if (isMissingValue(d)) {
+          renderMissingValue(ctx, col.getWidth(), context.rowHeight(j));
```

Comment:
shouldn't it be `renderMissingValue(ctx, cellDimension, rowHeight, x, padding + offset)`